ggshield/output/text/message.py

```diff
@@ -222,7 +222,9 @@ def iac_vulnerability_header(issue_n: int, vulnerability: IaCVulnerability) -> s
         issue_n,
         format_text("IaC", STYLE["detector"]),
         format_text(vulnerability.component, STYLE["detector"]),
-        format_text(vulnerability.policy, STYLE["detector"]),
+        format_text(
```

Comment:
I think it'd be better to change the format  template: add ` ({})` to the templatre, as well as an argument dedicated to policy_id (we might want to change the color of the id only, plus having white parenthesis allow to make it stand out a bit more.

Reply:
Much better like this indeed, thanks